docs/reference/cli/login.md

```diff
@@ -19,6 +19,6 @@ Authenticate your CLI with a Console installation
     $ convox login console.convox.com
     Authenticating with console.convox.com... OK
 
-    $ convox login console.convox.com -t a1234567-acde-1234-abcde-123abc456def
+    $ convox login console.convox.com -p a1234567-acde-1234-abcde-123abc456def
```

Comment:
sorry is it intended?

Reply:
`-p` is incorrect, it's `-t`

Reply:
hmmm I had a note to change this because I was working with schibsted and -t didn't work but -p did.. I will go back and look at my notes

Reply:
`-p` is for the V2 CLI, for v3 is `-t`